src/main/java/com/uwetrottmann/tmdb/services/MoviesService.java

```diff
@@ -104,6 +93,19 @@ Releases releases(
     );
 
     /**
+     * Get the release dates, certifications and related information by country for a specific movie id.
+     *
+     * The results are keyed by iso_3166_1 code and contain a type value which on our system, maps to:
+     * TYPE_PREMIERE, TYPE_THEATRICAL_LIMITED, TYPE_THEATRICAL, TYPE_DIGITAL, TYPE_PHYSICAL, TYPE_TV
```

Comment:
To link to other code in javadoc use for example for constants `{@link ReleaseDate#TYPE_PREMIERE}`.